app/models/order_import.rb

```diff
@@ -131,8 +131,16 @@ def handle_save_nothing_on_error # TODO refactor and rename
   end
 
   def import_row(row) # TODO refactor
-    row_importer = OrderRowImporter.new(row, self)
-    row_importer.import
+    begin
+      row_importer = OrderRowImporter.new(row, self)
+      row_importer.import
+    rescue => e
```

Comment:
Does this get triggered for invalid headers? I don't want to get an email every time there's bad user input.